### PR TITLE
add logging to appboy

### PR DESCRIPTION
**Pull request by stanishev**
add logging to appboy
also default local_timezone to false.

### CODE REVIEW

**Comment by stanishev** [comment]
@mikejaffe 
this PR is to merge your changes and mine into master of StreetEasy/appboy so we can start referencing the appboy gem from our se/Gemfile directly, without using commit hashes.

File: lib/appboy/rest/schedule_messages.rb [hunk 1]
```diff
@@ -3,14 +3,16 @@ module REST
     class ScheduleMessages < Base
       attr_reader :app_group_id, :send_at, :messages, :segment_id, :local_timezone, :external_user_ids, :campaign_id
 
-      def initialize(app_group_id, send_at:, messages: [], external_user_ids: [], campaign_id: nil, segment_id: nil,  local_timezone: true)
```
Comment (akshayarora) [inline lib/appboy/rest/schedule_messages.rb]: I assume it's okay to change the order of parameters since we reference the parameter when calling?

Comment (stanishev) [inline lib/appboy/rest/schedule_messages.rb]: yes, I had the same question when I was reordering the params and tried it out.

Comment (akshayarora) [inline lib/appboy/rest/schedule_messages.rb]: Not something you need to change obviously, but then I question why were the parameters re-ordered at all? It is entirely possible to make a call without specifying.

**Comment by mikejaffe** [comment]
Assuming reordering params works for you, I think this looks ok.

File: lib/appboy/rest/schedule_messages.rb (hunk 1 above)
Comment (stanishev) [inline lib/appboy/rest/schedule_messages.rb]: sure, I did it a bit on a whim, I grouped the parameters that we care about and use to come first.

segment_id for example we don't use.

**Pull request merged**